example/spec/parallel/ingest/ingestFromPdrSpec.js

```diff
@@ -141,10 +142,12 @@ describe('Ingesting from PDR', () => {
 
   afterAll(async () => {
     // clean up stack state added by test
-    await deleteGranule({
-      prefix: config.stackName,
-      granuleId: testDataGranuleId,
-    });
+    await Promise.all(ingestGranuleExecution.originalPayload.granules.map(
```

Comment:
Should we consider a promise.AllSettled here with an eval/throw?

Reply:
Also this is relying on the test at L433 to succeed to get the cleanup information.... which is eventually waiting on the `queueGranulesOutput` call at L296.....

I agree with the change here (reducing the over-agressive cleanup) in theory, but I'm wondering if this shouldn't be relying directly on `queueGranulesOutput.payload` or perhaps a more broad database query based on the test suffix rather than relying on chained test outputs for the input to generate the granule list.

Given this PR was to resolve the API fails due to the agreed-upon change and/or this is an improvement + better than a try/catch/ignore pattern, :thumbsup: for now, but we should proably talk more generally about the pattern.